next/components/forms/FieldErrorMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import cx from 'classnames'
+import React, { DOMAttributes, FC } from 'react'
+
+interface FieldErrorMessageProps {
+  errorMessage?: string
+  errorMessageProps?: DOMAttributes<never>
+}
+
+const FieldErrorMessage: FC<FieldErrorMessageProps> = ({errorMessage, errorMessageProps}) => {
+  const errorStyle = cx(
+    'mt-1 text-sm text-error'
+  )
+
+  return errorMessage ? (
+    <div className={errorStyle} {...errorMessageProps}>
```

Comment:
no need to make this more verbous than it needs to be

```suggestion
  return errorMessage ? (
    <div className="mt-1 text-sm text-error" {...errorMessageProps}>
```